infra/Pos/Subscription/Dns.hs

```diff
@@ -44,10 +45,15 @@ type KnownRelays = Map NodeId KnownRelay
 activeRelays :: KnownRelays -> [NodeId]
 activeRelays = map fst . filter (relayActive . snd) . M.toList
 
+-- TODO: Use valency and fallbacks
```

Comment:
Oh right, it becomes significantly more complicated if you want to hold onto subscriptions as long as possible... although, for wallets, there's no changing set of "best peers" so it's easier.